agent/fw_drupal8.c

```diff
@@ -366,6 +366,70 @@ NR_PHP_WRAPPER(nr_drupal8_post_implements_hook) {
 }
 NR_PHP_WRAPPER_END
 
+/*
+ * Purpose : Handles ModuleHandlerInterface::invokeAllWith()'s callback
+ *           and ensure that the relevant module_hook function is instrumented.
+ */
+NR_PHP_WRAPPER(nr_drupal9_invoke_all_with_callback) {
+  zval* module = NULL;
+
+  (void)wraprec;
+
+  NR_PHP_WRAPPER_REQUIRE_FRAMEWORK(NR_FW_DRUPAL8);
+
+  module = nr_php_arg_get(2, NR_EXECUTE_ORIG_ARGS TSRMLS_CC);
+  if (!nr_php_is_zval_non_empty_string(module)) {
+      NR_PHP_WRAPPER_CALL;
```

Comment:
Could consider putting NR_PHP_WRAPPER_CALL here and [here](https://github.com/newrelic/newrelic-php-agent/pull/665/files#diff-6c636f6e3733aae919f4c4cf66ec5226065346587de8cbaf3918f483f9d0f523R390) in the `leave` block.

Reply:
fixed in [c67d9a2](https://github.com/newrelic/newrelic-php-agent/pull/665/commits/c67d9a2562a488ac99ca8aa925afc65e47c58eec)